USAGE.adoc

```diff
@@ -66,22 +66,83 @@ Specify class names in their fully qualified form (i.e., inner classes should be
 
+==== For Fields
+
 The command line flag `-F filename.txt` or `--additionalFields filename.txt` allows you to find specific field usages.
 The syntax here is the same as for specifying method names.
-Use the additional flag `-i` or `--onlyIncludeSpecified` to only show usage of specified classes, methods, and fields.
 
-=== Finding Usage by Package
+Example:
+
+[source]
+----
+jenkins.model.Jenkins#ADMINISTER
+----
+
+==== For Classes
+
+The command line flag `-C filename.txt` or `--additionalClasses filename.txt` allows you to find specific classes usages.
+The syntax here is the same as for specifying method names.
+
+Example:
+
+[source]
+----
+hudson.model.Hudson
+----
+
+==== Finding Usage by Package
 
-By using just a package name instead of the fully qualified class name in any of the options above, this searches for classes that start with the same specified package.
+By using just a package name instead of the fully qualified class name in "Class", "Method" or "Field" options, this searches for classes that start with the same specified package.
 For example, the following snippet would search for usage of methods or fields named `toString` in any classes inside the `hudson.util` package or its descendants.
-Note the trailing `.` before the `#` separator which makes the search more specific to `hudson.util` rather than also including the hypothetical package `hudson.utils` or `hudson.utility`.
+
+By default, the package name is check for exactness, but you can customize it by adding `*` at the beginning, the end or both,
+to have a `startsWith`, `endsWith` or `contains` behavior respectively.
 
 [source]
 ----
-hudson.util.#toString
+hudson.util.*#toString
 ----
 
+Note the trailing `.*` before the `#` separator which makes the search more specific to `hudson.util` rather than also including the hypothetical package `hudson.utils` or `hudson.utility`.
+
+==== Limit scope of plugins
+By default all the plugins included in the update-center JSON will be analysed.
+You can use the arguments `-l filename.txt` or `--limitPlugins filename.txt` to limit which plugins you can want to analyse.
+The file must contain the artifact ID of the plugins, one per line.
+
+==== Additional information
+
+Multiple entries can be put in the file related to those options, one per line.
+The lines are trimmed and if one starts with a `#`, it's considered as a comment, and thus ignored.
+The "class full name" can be written with `.` or with `/` without incidence.
+
 === Configuring Parallelism
 
 The default parallelism settings will allow up to 4x the number of available CPUs downloads at a time.
 This can be overridden by the `-D N` or `--downloadConcurrent=N` option for some number `N`.
+
+=== Including libraries from plugins
+
+The plugins often include libraries in addition to their direct code. By default they are not included.
+This can be overridden by the `-p` or `--includePluginLibs=N`.
```

Comment:
```suggestion
This can be overridden by the `-p` or `--includePluginLibs=true`.
```

https://github.com/kohsuke/args4j/blob/dc2e7e265caf15a1a146e3389c1f16a8781f06cf/args4j/src/org/kohsuke/args4j/spi/BooleanOptionHandler.java#L16-L17

Reply:
Good catch 👍, bad copy-paste from the concurrent :p